packages/account/src/Constants/poo-identifier.ts

```diff
@@ -0,0 +1,23 @@
+export const IDENTIFIER_TYPES = Object.freeze({
+    ACCOUNT_ID: 'account_id',
+    ACCOUNT_NUMBER: 'account_number',
+    BANK_ACCOUNT_NUMBER: 'bank_account_number',
+    CARD_NUMBER: 'card_number',
+    EMAIL_ADDRESS: 'email_address',
+    MOBILE_NUMBER: 'mobile_number',
+    USER_ID: 'user_id',
+});
+
+export const POO_STATUSES = {
+    NONE: 'none',
+    PENDING: 'pending',
+    REJECTED: 'rejected',
+    VERIFIED: 'verified',
+};
```

Comment:
Since these statuses are similar to POI POA, thinking if we can move this to constant file notspecifically for Proof of ownership